doc/readme/README_linalg.md

```diff
@@ -0,0 +1,190 @@
+## Internal linear algebra library
```

Comment:
It was modified from the old [README](https://github.com/shogun-toolbox/shogun/wiki/README_linalg) but I was not sure where it was located and I don't want to edit it directly